chromadb/db/clickhouse.py

```diff
@@ -155,27 +160,28 @@ def get_collection(self, name):
     def list_collections(self) -> Sequence[Sequence[str]]:
         return self._get_conn().query(f"""SELECT * FROM collections""").result_rows
 
-    def update_collection(self, current_name, new_name, new_metadata):
+    def update_collection(
+        self, current_name: str, new_name: Optional[str] = None, new_metadata: Optional[Dict] = None
+    ):
         if new_name is None:
             new_name = current_name
         if new_metadata is None:
-            new_metadata = self.get_collection(current_name)[0]
+            new_metadata = self.get_collection(current_name)[0][2]
```

Comment:
get collection returns a list of each row as a tuple, this correctly fetches the metadata

cc @jeffchuber